### PR TITLE
Add rules_cc_hdrs_map 0.0.3

### PR DESCRIPTION
**Pull request by AleksanderGondek**
Introduce rules extending the cc compilation capabilities with hdrs maps

### CODE REVIEW

File: modules/rules_cc_hdrs_map/0.0.1/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    build_targets:
+    - '@rules_cc_hdrs_map//...'
```
Comment (alexeagle) [inline modules/rules_cc_hdrs_map/0.0.1/presubmit.yml:13]: you probably don't want `//...` here as that descends into packages that load stardoc, and are therefore not part of the public API

Comment (AleksanderGondek) [reply modules/rules_cc_hdrs_map/0.0.1/presubmit.yml]: Thanks for the tip!

**Comment by meteorcloudy** [comment]
I see you already have the examples directory: https://github.com/AleksanderGondek/rules_cc_hdrs_map/blob/master/.github/workflows/ci.yaml#L67

Maybe you also want to add that as a test_module?

See an example here: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/rules_jni/0.6.1/presubmit.yml#L11-L28

**Comment by keith** [comment]
failures here appear to be real

**Comment by alexeagle** [comment]
Hey @AleksanderGondek should we close this one? It has been a year since we heard from you here.

**Comment by AleksanderGondek** [comment]
I have been working in completely different areas and dropped the ball on this one. 
I intend to get back into it in upcoming weeks but I will be slow burning - I do not mind opening another Pull Request when I am ready, if closing this one for the moment helps.
Thank you for checking with me!

**Comment by Wyverald** [comment]
Hello! I'm doing some routine cleanup of stale PRs. If you're still working on this and planning to make progress soon, please let me know and we can re-open it.

**Pull request closed without merging**